machinelearning/pytorch.md

```diff
@@ -0,0 +1,217 @@
+# ML inference on Graviton CPUs with PyTorch
+
+**Introduction**
+
+PyTorch is an open-source machine learning framework based on the Torch library, used for applications such as computer vision and natural language processing. It can be used across training and inference of deep neural networks. This document covers how to use PyTorch based machine learning inference on Graviton CPUs, what runtime configurations are important and how to debug any performance issues. The document also covers instructions for source builds and how to enable some of the downstream features.
+
+# How to use PyTorch on Graviton CPUs
+There are multiple levels of software package abstractions available: Python wheel (easiest option but not the most performant one), Docker container (supports all the optimizations: onednn+acl backend, xla, additional packages and benchmarks), DLAMI (Deep Learning Amazon Machine Image that comes with all the optimizations that the docker container supports). Examples of using each method are below.
+
+**Using Python wheel**
+
+PyTorch wheel supports OpenBLAS backend for Graviton CPUs. This is not the most performant option yet. For the best performance, please use either the docker container or the dlami option descibed below.
```

Comment:
Maybe we should say:

To get started developing on Graviton, you can use the pytorch python wheel directly.  We recommend using the docker containers or AWS DLAMIs to achieve the best performance from your application.